db/db_impl/db_impl_secondary.h

```diff
@@ -17,18 +17,15 @@ namespace ROCKSDB_NAMESPACE {
 // A wrapper class to hold log reader, log reporter, log status.
 class LogReaderContainer {
  public:
-  LogReaderContainer()
-      : reader_(nullptr), reporter_(nullptr), status_(nullptr) {}
+  LogReaderContainer() : reader_(nullptr), reporter_(nullptr) {}
   LogReaderContainer(Env* env, std::shared_ptr<Logger> info_log,
                      std::string fname,
                      std::unique_ptr<SequentialFileReader>&& file_reader,
                      uint64_t log_number) {
     LogReporter* reporter = new LogReporter();
-    status_ = new Status();
     reporter->env = env;
     reporter->info_log = info_log.get();
     reporter->fname = std::move(fname);
-    reporter->status = status_;
```

Comment:
The status is updated by LogReporter in line 62 below. Should we update RecoverLogFiles() to check LogReaderContainer's status?